pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -82,14 +83,15 @@ func NewCertRotationController(
 	certRotator, err := certrotation.NewCertRotationController(
 		"AggregatorProxyClientCert",
 		certrotation.SigningRotation{
-			Namespace:     operatorclient.OperatorNamespace,
-			Name:          "aggregator-client-signer",
-			Validity:      30 * defaultRotationDay,
-			Refresh:       15 * defaultRotationDay,
-			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets(),
-			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets().Lister(),
-			Client:        kubeClient.CoreV1(),
-			EventRecorder: eventRecorder,
+			Namespace:              operatorclient.OperatorNamespace,
+			Name:                   "aggregator-client-signer",
+			Validity:               30 * defaultRotationDay,
+			Refresh:                30 * defaultRotationDay,
```

Comment:
Why you're changing the refresh rate?

Reply:
this was just a test before I changed library-go with an explicit option - need to clean it up now